src/components/Reports/FinancialAccountsReport/FinancialAccounts/FinancialAccounts.tsx

```diff
@@ -17,43 +11,40 @@ import {
 import { useFetchAllPages } from 'src/hooks/useFetchAllPages';
 import { useLocale } from 'src/hooks/useLocale';
 import { currencyFormat } from 'src/lib/intlFormat';
-import { AccountsList as List } from '../AccountsListLayout/List/List';
+import { AccountsList as List } from '../../AccountsListLayout/List/List';
+import {
+  FinancialAccountContext,
+  FinancialAccountType,
+} from '../Context/FinancialAccountsContext';
 import { useEntryHistoriesQuery } from './GetEntryHistories.generated';
 import {
   FinancialAccountsDocument,
   FinancialAccountsQuery,
   useFinancialAccountsQuery,
 } from './GetFinancialAccounts.generated';
 import { useSetActiveFinancialAccountMutation } from './SetActiveFinancialAccount.generated';
```

Comment:
Is there a reason to not put these three queries in the same file? I like having one `.graphql` file per component vs one file per operation.